File-Formats/PDF/Working-with-DigitalSignature.md

```diff
@@ -2786,46 +2465,40 @@ The Essential PDF supports creating long term signature validation when signing
 
 N> The resulted PDF document size will be large since all the necessary signature information, Certificate Revocation List (CRL), and Online Certificate Status Protocol (OCSP) are embedded.
 
-The following code example explains how to create LTV PDF.
+The following code example explains how to create LTV PDF using [PdfSignature](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignature.html) class.
```

Comment:
he following code example explains how to create LTV using [EnableLtv]() property in [PdfSignature]() class when signing the PDF document.

Reply:
Changed the content